src/setup/registerNodes.js

```diff
@@ -73,7 +74,6 @@ const customParserFactory = (nodeType, primaryIdentifier, secondaryIdentifier) =
 };
 
 export default function registerNodes({ registerNode, registerBpmnExtension }) {
-  registerNode(startEvent);
```

Comment:
Adding the start event first is no longer necessary since we explicitly sort the controls in the left hand controls column.